services/workflows-service/src/common/utils/unified-api-client/unified-api-client.ts

```diff
@@ -14,8 +15,29 @@ export type TReportRequest = Array<{
   businessReportId?: string;
   withQualityControl?: boolean;
 }>;
+export type TOcrImages = Array<
+  | {
+      remote: {
+        imageUri: string;
+        mimeType: string;
+      };
+    }
+  | {
+      base64: string;
+    }
+>;
 
-export class UnifiedApiClient {
+interface TOcrImage {
+  runOcr({
+    images,
+    schema,
+  }: {
+    images: TOcrImages;
+    schema: TSchema;
+  }): Promise<axios.AxiosResponse<any>>;
+}
```

Comment:
What is the goal of creating and using this interface over implementing runOcr directly?